File: 1159.cpp
#include<iostream>
using namespace std;
int main()
{
    int x, i, j, a, sum;
    while(1)
    {
        cin>>x;
        sum = 0;
        if(x==0)
            break;
        else if(x%2==0)
        {
            for(j = 1; j<=5; j++)
            {
                sum = sum + x;
                x+=2;
            }
            cout<<sum<<endl;
        }
        else
        {
            a = x+1;
            for(j = 1; j<=5; j++)
            {
                sum = sum + a;
                a+=2;
            }
            cout<<sum<<endl;
        }
    }

    return 0;
}

File: 1159.py
while(True):
    x = int(input())
    sum = 0
    if x==0:
        break
    elif x%2==0:
        for i in range(1, 6):
            sum = sum + x
            x = x+2
        print(sum)
    else:
        a = x+1
        for i in range(1, 6):
            sum = sum + a
            a = a+2
        print(sum)

